src/components/shared/Dropdown.css

```diff
@@ -10,17 +10,19 @@
   z-index: 1000;
 }
 
+.dropdown-block {
+  align-self: flex-end;
```

Comment:
`position: relative` is needed so that the absolutely positioned dropdown list renders fine